### ISSUE TITLE
Loading PAC from file

### ISSUE THREAD

**Issue by genotrance** [carsonyl/pypac#18]
My VPN adds an AutoConfigURL to the registry with the file:/// format. Since os.path.isfile() doesn't recognize this as a file, pypac fails in api.py and doesn't load the PAC file.

It will be great if pypac could identify such file:/// URLs and still load them.

Further, my VPN's PAC path is file:///Users\\username\... so it even skips the drive letter on Windows so file:/// should be replaced with \\.

**Comment by genotrance** [comment]
Here's how I'm working around it if it helps:-

```python
    acu = pypac.windows.autoconfig_url_from_registry()
    if acu:
        acu = acu.replace("file:///", "")
        if ":" not in acu:
            acu = "c:\\" + acu
        if os.path.isfile(acu):
            with open(acu, "r") as f:
                pac = pypac.parser.PACFile(f.read())
```

**Comment by carsonyl** [comment]
I've released 0.8.0, which should add this functionality. Please let me know if it works for you.

Out of curiosity, what VPN software you using? Does it always set file:// PAC URLs?

**Comment by genotrance** [comment]
Awesome, thanks - will try it out soon.

We use SonicWall and it downloads a PAC file itself and adds a file:// to AutoConfigURL.
